PINCache/PINDiskCache.m

```diff
@@ -305,17 +303,11 @@ + (BOOL)moveItemAtURLToTrash:(NSURL *)itemURL
 + (void)emptyTrash
 {
     dispatch_async([self sharedTrashQueue], ^{
-        NSError *searchTrashedItemsError = nil;
-        NSArray *trashedItems = [[NSFileManager defaultManager] contentsOfDirectoryAtURL:[self sharedTrashURL]
-                                                              includingPropertiesForKeys:nil
-                                                                                 options:0
-                                                                                   error:&searchTrashedItemsError];
-        PINDiskCacheError(searchTrashedItemsError);
-        
-        for (NSURL *trashedItemURL in trashedItems) {
+        if (_sharedTrashURL != nil) {
             NSError *removeTrashedItemError = nil;
-            [[NSFileManager defaultManager] removeItemAtURL:trashedItemURL error:&removeTrashedItemError];
+            [[NSFileManager defaultManager] removeItemAtURL:_sharedTrashURL error:&removeTrashedItemError];
```

Comment:
`[NSFileManager defaultManager]` is not thread safe. We should create on in here before using it.